src/migrations/1677479564056-sync.ts

```diff
@@ -1,21 +1,25 @@
 import { MigrationInterface, QueryRunner } from "typeorm";
 
-export class createTableMatchingEvent1674651604338 implements MigrationInterface {
```

Comment:
why the new migrations are creating user table again?

Reply:
Done